doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -24,6 +24,20 @@ You can configure the Matter protocol to use NFC tag for commissioning, instead
 
 To enable NFC for commissioning and share the onboarding payload in an NFC tag, set the :kconfig:option:`CONFIG_CHIP_NFC_COMMISSIONING` Kconfig option.
 
+.. _ug_matter_configuring_optional_ble_arbiter:
+
+Advertising arbiter for Bluetooth® LE services
+==============================================
+
+You can implement the Bluetooth LE Arbiter class to enable easier coexistence of application components that want to advertise their Bluetooth LE services, for example for your custom services during :ref:`ug_matter_network_topologies_commissioning_stages_discovery`.
+
+When several application services are active, they can define a request to the arbiter with the desired priority.
+If the service with the highest priority stops advertising, the arbiter automatically selects the next service in the queue.
+If the service that requests advertising has a higher priority than the service that is running when the request is submitted, the advertising is restarted using parameters defined in the new request.
+
+The class was created for the Zephyr platform in the Matter SDK.
+For more information about what the class offers, see the :file:`BLEAdvertisingArbiter.h` file in the `Zephyr platform source files`_ in Matter upstream.
```

Comment:
@LuDuda , when you created the task, did you have in mind where this information should go? Currently, it's placed in "Advanced Kconfigs" page, but obviously we're not discussing any Kconfigs here, so either the name of the page must change or we must move the section.